src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -141,13 +152,68 @@ your domain. Be sure to include the escaping characters.
 </string>
 ```
 
-### Test for the presence of your native app {: #test-native }
+Finally, publish your updated Android app to the Play store.
+
+### Define the relationship to an installed PWA {: #relationship-web }
+
+{% Aside %}
+**Coming Soon!**
+Expected in Chrome 81, websites can check if their PWA, served from the same
+domain is installed.
+Microsoft is actively working on enabling this API for Edge and we hope to
+see it land around Edge 82.
+{% endAside %}
+
+To define the relationship to an installed PWA, add a `related_applications`
+entry in the web app manifest, set `"platform": "webapp"` and provide
+the full path to the PWAs web app manifest in the `url` property.
+
+```json/3-4
+{
+  …
+  "related_applications": [{
+    "platform": "webapp",
+    "url": "https://app.example.com/manifest.json",
+  }],
+  …
+}
+```
+
+#### PWA served from a different domain
+
+{% Aside %}
+**Coming Soon!**
+Starting in Chrome 82, websites can check if their PWA, served from the same,
+or different domain is installed.
```

Comment:
```suggestion
In addition to detecting if their PWA is served from the same domain, 
Chrome 82 lets websites check if their PWA is served from a different domain.
```

Reply:
I thought, "didn't I just read this" and had to scroll back up to see what's different. Yes, my suggestions violates "Express coordinate ideas in similar form." However, breaking the rule to clarify what's different.